File: assignments/a1/q9/Python/Skp.py
#Skp:top#
#Skp:import#

# <rule>:Skp ::= <SKIP>
class Skp(Rule): #Skp:class#

    className = "Skp"
    ruleString = "<rule>:Skp ::= <SKIP>"
    skip = None

    def __init__(skip):
        #Skp:init#
        self.skip = skip

#Skp#

File: assignments/a1/q8/Java/Skp.java
//Skp:top//
//Skp:import//
import java.util.*;

// <rule>:Skp ::= <SKIP>
public class Skp extends Rule /*Skp:class*/ {

    public static final String $className = "Skp";
    public static final String $ruleString =
        "<rule>:Skp ::= <SKIP>";

    public Token skip;

    public Skp(Token skip) {
//Skp:init//
        this.skip = skip;
    }

    public static Skp parse(Scan scn$, Trace trace$) {
        if (trace$ != null)
            trace$ = trace$.nonterm("<rule>:Skp", scn$.lno);
        Token skip = scn$.match(Token.Match.SKIP, trace$);
        return new Skp(skip);
    }

//Skp//
}
